package.json

```diff
@@ -15,20 +15,32 @@
   "scripts": {
     "build:ts": "tsc -b",
     "commit": "npx git-cz",
-    "diff_versions": "ts-node scripts/diff_versions",
+    "diff_versions": "ts-node scripts/diff_versions.ts",
     "semantic-release": "semantic-release",
     "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
-    "format": "cross-env prettier-eslint $PWD\"/src/**/*.{ts,tsx,js,jsx,json,css,md}\" --write"
+    "format": "cross-env prettier-eslint $PWD\"/src/**/*.{ts,tsx,js,jsx,json,css,md}\" --write",
+    "generate:defs": "node --loader @polkadot/dev-ts node_modules/.bin/polkadot-types-from-defs --endpoint metadata.json --package . --input ./src/polkadot",
+    "generate:meta": "node --loader @polkadot/dev-ts node_modules/.bin/polkadot-types-from-chain --endpoint metadata.json --package . --output ./src/polkadot --strict",
+    "fetch:definitions": "ts-node scripts/fetchDefinitions.ts",
+    "generate:consts": "ts-node scripts/generateConsts.ts",
+    "generate:types": "rimraf src/polkadot/* && yarn fetch:definitions && yarn generate:defs && yarn generate:meta && yarn generate:consts && yarn format"
```

Comment:
Was there a reason the folder was changed from generate-types to polkadot? I know the sdk previously use the polkadot folder name but the name in this context doesn't make sense to me. If you want to keep it as `polkadot` you need to fix the Readme to specify the correct path when importing types.

Reply:
I liked `polkadot` to add in context that types in this folder were related to something. 

Anyone using the types from `polymesh-types` will also see paths like `@polymeshassociation/polymesh-types/polkadot/...` which give context as well that we are import some polkadot related types from polymesh-types. 

And this also adds consistency in SDK repo as well with current imports. 

I wanted to keep the `generated` folder to the ones that we generate separate from the typescript types for polkadot